tasks/util.py

```diff
@@ -1231,7 +1231,7 @@ def output(self):
         table lives in a special-purpose schema in Postgres derived using
         :func:`~.util.classpath`.
         '''
-        return PostgresTarget(classpath(self), self.task_id)
+        return PostgresTarget(classpath(self), self.task_id.split('.')[-1])
```

Comment:
Just to be sure, this is now need because of the auto_namespace option activated isn't it?

Reply:
Yes. It's not actually needed, but this makes the tables inside the schema keep the same name, which means we don't have to reimport them.